tracer/test/test-applications/debugger/dependency-libs/Samples.Probes.TestRuns/Shared/InterfaceTesting.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Globalization;
+
+namespace Samples.Probes.TestRuns.Shared
+{
+    public interface IGenericInterface<T>
+    {
+        public T Value { get; set; }
+    }
+
+    public interface IInterface
+    {
+#if NETCOREAPP3_1_OR_GREATER
```

Comment:
Why do we limit this to `NETCOREAPP3_1_OR_GREATER`?

Reply:
Fixed in 963a0d8f0b02952b982c82fd16239363573c7b19